src/screens/Maxes/Maxes.tsx

```diff
@@ -67,97 +109,71 @@ const Maxes: React.FC = () => {
     }
   };
```

Comment:
This function doesn't work, and seems like it now belongs in `use-storage.hook.ts`